src/components/home/WhyJoin.jsx

```diff
@@ -1,5 +1,35 @@
+import Title from "../Title";
+import { FOCUS } from "@/data/focus";
+import Focus from "./Focus";
+import Picture from "../Picture";
 const WhyJoin = () => {
-  return <div>WhyJoin</div>;
+  return (
+    <div className="p-16 flex flex-col items-center">
```

Comment:
if it is just for vertical padding, try py-16 and for the horizontal you can use width to style it so it can be more responsive.